src/dialects/db2/data-types.js

```diff
@@ -265,6 +265,19 @@ module.exports = BaseTypes => {
     }
   }
 
+  class TIME extends BaseTypes.TIME {
+    constructor() {
+      super();
+
+      if (this._zoned) {
+        warn('DB2 does not support TIME WITH TIMEZONE. Plain `TIME` will be used instead.');
+        this._zoned = undefined;
+      }
+    }
+  }
+
+  delete TIME.parse;
```

Comment:
I hadn't noticed this line before, what is the reason for it?

Reply:
In the beginning, I don't understand everything, so I took as an example what you sent first as a guide for my function request 

https://github.com/sequelize/sequelize/blob/85b0984c137dcc4c79ae5627ac7973e589b0150c/src/dialects/postgres/data-types.js#L313

Reply:
Maybe I should remove this

Reply:
Yes, I don't think it does anything as there doesn't seem to be a static `parse` method present in its prototype chain